src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js

```diff
@@ -10,16 +10,28 @@ export default class DocumentItemBody extends Component {
   }
 
   itemStatus = (item) => ({
-    IsOnShelf: () =>
-      invenioConfig.ITEMS.canCirculateStatuses.includes(item.status) &&
-      !item.circulation,
+    isOnCirculation: () =>
```

Comment:
Let's keep the name we normally use:
```suggestion
    canCirculate: () =>
```